python/playback_label/playback_label.py

```diff
@@ -70,25 +71,46 @@ def set_shotgun_data(self, sg_data):
             if sg_data.get("sg_uploaded_movie"):
                 self._playable = True
         
-        if self._playable:
+        if self.playable and self.interactive:
             self.setCursor(QtCore.Qt.PointingHandCursor)
         else:
             self.unsetCursor() 
         
     @property    
-    def playbable(self):
+    def playable(self):
         """
-        Returns True if the label is playbable given its current Shotgun data.
+        Returns True if the label is playable given its current Shotgun data.
         """
         return self._playable
-            
+
+    def _get_interactive(self):
+        """
+        Whether a playable label is interactive. If it is not, then the play
+        icon will not be overlayed on the thumbnail image, and the playback
+        signal will not be emitted on click event.
+        """
+        return self._interactive
+
+    def _set_interactive(self, state):
+        self._interactive = bool(state)
+
+        if not self._interactive:
+            self.unsetCursor()
+        elif self.playable:
+            self.setCursor(QtCore.Qt.PointingHandCursor)
```

Comment:
If interactive and not playable this'll leave the cursor as is.  Is that the intended behavior?